spaces/domains.json

```
@@ -322,5 +322,6 @@
   "vote.meebitsdao.com": "meebitsdao.eth",
   "vote.dopeshibas.fun": "dopeshibas.eth",
   "vote.manifoldfinance.com": "manifoldfinance.eth",
-  "grant.mint.club": "mintgrant.eth"
+  "grant.mint.club": "mintgrant.eth",
+  "vote.minto.finance": "minto.eth",
```

Please remove the conma at theend of the line.
```suggestion
  "vote.minto.finance": "minto.eth"
```